### DISCUSSION TITLE
23.11.24 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#78] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 발표 준비
    - 우재님 코드 리뷰 (회의 이후!)
- 내일은 어떤 일을 하나요?
    - 서비스 테스트 작성

**Comment by Jeonwoojae** [comment]
- 내일은 어떤 일을 하나요?
  - 플레이리스트 음악 저장 기능 구현
  - 플레이리스트 알림 기능 연동
  - 친구 추가 알림 기능 연동
  - 코드 많이 개선하기

**Comment by Nine-JH** [comment]
* 오늘은 어떤 일을 했나요?
  * 연관관계 테스트 완료!
* 내일은 어떤 일을 하나요?
  * 통합 테스트 수행
